shell/middleware/authenticated.js

```diff
@@ -146,41 +143,8 @@ function invalidResource(store, to, redirect) {
 }
 
 export default async function({
-  route, app, store, redirect, $cookies, req, isDev, from, $plugin, next
+  route, store, redirect, from, $plugin, next
 }) {
-  if ( route.path && typeof route.path === 'string') {
-    // Ignore webpack hot module reload requests
-    if ( route.path.startsWith('/__webpack_hmr/') ) {
-      return;
-    }
```

Comment:
This was not needed. I believe it was an artifact of ssr. You can see that hmr is working fine.


https://github.com/rancher/dashboard/assets/55104481/4c517f4a-6a05-4cf3-9a51-8291743b1298